hypothesis-python/tests/cover/test_conjecture_engine.py

```diff
@@ -1512,3 +1381,52 @@ def f(data):
     shrinker.adaptive_example_deletion()
 
     assert shrinker.remove_discarded.call_count == 1
+
+
+def shrinking_from(start):
```

Comment:
I'm much happier with this new style of test than the previous monkeypatching based style. I'll probably be gradually moving the rest of the tests over to it, and will write new shrinker tests in this style.